variables.tf

```diff
@@ -27,6 +27,7 @@ variable "additional_tfe_workspaces" {
     file_triggers_enabled          = optional(bool, true)
     global_remote_state            = optional(bool, false)
     name                           = optional(string, null)
+    permissions_boundary_arn       = optional(string, null)
```

Comment:
this is old?

Reply:
No, still present. Is this variable not required since we pass it to workpace as 
`permissions_boundary_arn       = aws_iam_policy.workspace_boundary[0].arn`

Reply:
@svashisht03 no you are not passing down this variable, you are referencing a resource
`permissions_boundary_arn = try(aws_iam_policy.workspace_boundary[0].arn, null)`.
This var is now not doing anything, can you remove this var? and also the same var in tfe_workspace?